sources/platform/actors/development/actor_definition/input_schema/specification.md

```diff
@@ -183,16 +183,18 @@ Rendered input:
+| `maxLength`  | Integer                                                                                                                                                                                                               | No                                         | Maximum length of the string.                                                                                                       |
+| `enum`       | [String]                                                                                                                                                                                                              | Required if <br/>`editor` <br/>is `select` | Using this field, you can limit values <br/>to the given array of strings. <br/>Input will be displayed as select box.              |
+| `enumTitles` | [String]                                                                                                                                                                                                              | No                                         | Titles for the `enum` keys described.                                                                                               |
+| `nullable`   | Boolean                                                                                                                                                                                                               | No                                         | Specifies whether `null` <br/>is an allowed value.                                                                                  |
+| `isSecret`   | Boolean                                                                                                                                                                                                               | No                                         | Specifies whether the input field<br />will be stored encrypted.<br />Only available <br />with `textfield` and `textarea` editors. |
+| `isAbsolute` | Boolean                                                                                                                                                                                                               | No                                         | Used with `datepicker` editor for absolute date value picker in form `YYYY-MM-DD`. This defaults to `true`.                          |
+| `isRelative` | Boolean                                                                                                                                                                                                               | No                                         | Used with `datepicker` editor for relative input in form `+/- {number} {unit}` e.g. `+ 3 weeks`.                                    |
```

Comment:
They are missing what units are allowed.

Reply:
Done, added supported units and also mention that `isAbsolute` and `isRelative` are also supported together